devops/openshift/forms-deploy.yml

```diff
@@ -175,8 +175,8 @@ objects:
             resource:
               name: cpu
               target:
-                type: AverageValue
-                averageValue: "0.8"
+                type: Utilization
+                averageUtilization: 50
```

Comment:
why is it 50 for form ?

Reply:
As part of the load testing for forms, the value is deried as 50%. This is similar to the .8 as mentioned previously.